src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,31 @@
+package core.basesyntax;
+
+public class Circle extends Shape {
+
+    private double radius;
+
+    public Circle(String color, double radius) {
+        super(Colors.valueOf(color));
+        this.radius = Math.round(Math.abs(radius));
+    }
+
+    public double getRadius() {
+        return radius;
+    }
+
+    public void setRadius(double radius) {
+        this.radius = radius;
```

Comment:
Consider validating the radius in the `setRadius` method to ensure it is non-negative, similar to the constructor.